src/hooks/useContract.ts

```diff
@@ -342,13 +342,14 @@ interface AllContractsObject {
 }
 export function useAllContracts(): AllContractsObject | null {
```

Comment:
Can you remove this function too? It's only used in `useApproveAndSwap`, and only to get erc20 contracts for approval.